cmd/rpcdaemon/cli/config.go

```diff
@@ -496,22 +497,30 @@ func RemoteServices(ctx context.Context, cfg *httpcfg.HttpCfg, logger log.Logger
 	var remoteCE *remoteConsensusEngine
 
 	if cfg.WithDatadir {
-		// TODO: refactor to use if
 		switch {
 		case cc != nil:
 			switch {
-			case cc.Bor != nil && !polygonSync:
-				var borKv kv.RoDB
-
-				// bor (consensus) specific db
-				borDbPath := filepath.Join(cfg.DataDir, "bor")
-				logger.Warn("[rpc] Opening Bor db", "path", borDbPath)
-				borKv, err = kv2.NewMDBX(logger).Path(borDbPath).Label(kv.ConsensusDB).Accede().Open(ctx)
-				if err != nil {
-					return nil, nil, nil, nil, nil, nil, nil, ff, err
+			case cc.Bor != nil:
+				if polygonSync {
+					stateReceiverContractAddress := cc.Bor.GetStateReceiverContract()
+					bridgeReader, err = bridge.AssembleReader(ctx, cfg.DataDir, logger, stateReceiverContractAddress)
+					if err != nil {
+						return nil, nil, nil, nil, nil, nil, nil, ff, nil, err
+					}
+					engine = ethash.NewFaker()
+				} else {
```

Comment:
@shohamc1 why the else? we still want to execute the code below and set engine = bor.NewRo

Reply:
It needs additional work (https://github.com/erigontech/erigon/issues/11171), but I guess we can keep it with a note that says it is partially supported right now.

Reply:
It is not a problem because we haven't removed the `bor.Snapshot` code. So the bor_* RPCs will work even with standalone as that table is still getting populated even with Astrid standalone. There is no need for a comment in my opinion.

Reply:
Once we remove `bor.Snapshot` code then there will be no longer duplication. Before we do that we will have to have the bor_* RPCs transitioned over to the new `Producers` API.

Reply:
In my testing, I found some RPCs like `bor_getCurrentValidators` don't work on standalone rpcdaemon when using an Astrid standalone datadir.

Reply:
Hmm, that is unexpected. It should work fine. But that's a whole other topic. Let's leave it for another day.